web/templates/blocks/hosts_table.html.tmpl

```diff
@@ -18,14 +20,10 @@
                     <td>
                         <a href="/clusters/{{ index .TrentoMeta "trento-ha-cluster" }}">{{ index .TrentoMeta "trento-ha-cluster" }}</a>
                     </td>
-                    <td>
-                        {{- range $Key, $Value := .TrentoMeta }}
-                            {{- if eq $Key "trento-ha-cluster" }}
-                            {{- else }}
-                                <span class='badge badge-pill badge-info'>{{ $Value }}</span>
-                            {{- end }}
-                        {{- end }}
-                    </td>
+
+                    <td>{{ .TrentoMetaKey "trento-sap-environment" }}</td>
```

Comment:
I don't know if we need this new `.TrentoMetaKey` method. In other parts of the code we just use `index .TrentoMeta "trento-ha-cluster"` to get the value (check the cluster entry above). `.TrentoMeta` already returns a map with all of the metadata entries, and you only need to use the `index` method.
In any case, use the same option in all of them